src/main/java/mate/academy/spring/dao/type/RoleNames.java

```diff
@@ -0,0 +1,5 @@
+package mate.academy.spring.dao.type;
+
+public enum RoleNames {
+    ADMIN, USER
+}
```

Comment:
```suggestion
public enum RoleName {
    ADMIN, USER
}
```
we suppose to make enum RoleName inside User class? and remove s in the end

Reply:
ok